exp/clamp/clamp.go

```diff
@@ -0,0 +1,26 @@
+package clamp
```

Comment:
maybe `x/exp/ordered` instead? (as currently all functions use `constraints.Ordered`) 🤔

Reply:
Ah yeah, that's better. Will adjust.